yarn.lock

```diff
@@ -8747,31 +8747,7 @@ gatsby-plugin-robots-txt@^1.6.8:
     "@babel/runtime" "^7.16.7"
     generate-robotstxt "^8.0.3"
 
-gatsby-plugin-sharp@4.4.0:
-  version "4.4.0"
-  resolved "https://registry.yarnpkg.com/gatsby-plugin-sharp/-/gatsby-plugin-sharp-4.4.0.tgz#d3d182f1397011aeb969f4300022ff13df81e720"
-  integrity sha512-X2Syc6YfOD2O+5A2Lrd/l/HXHIAIjcbRfP38uJPbG0cZg3xRd3T7RaDEcDrFylHvpYZfcqjzumfwPeQuhhxUUQ==
-  dependencies:
-    "@babel/runtime" "^7.15.4"
-    async "^3.2.2"
-    bluebird "^3.7.2"
-    filenamify "^4.3.0"
-    fs-extra "^10.0.0"
-    gatsby-core-utils "^3.4.0"
-    gatsby-plugin-utils "^2.4.0"
-    gatsby-telemetry "^3.4.0"
-    got "^11.8.3"
-    lodash "^4.17.21"
-    mini-svg-data-uri "^1.4.3"
-    potrace "^2.1.8"
-    probe-image-size "^6.0.0"
-    progress "^2.0.3"
-    semver "^7.3.5"
-    sharp "^0.29.3"
-    svgo "1.3.2"
-    uuid "3.4.0"
-
-gatsby-plugin-sharp@~4.19.0:
+gatsby-plugin-sharp@4.19.0, gatsby-plugin-sharp@4.4.0, gatsby-plugin-sharp@~4.19.0:
```

Comment:
i think these are just labels in the lock file, but this looks weird, i imagine it just needs to be `gatsby-plugin-sharp@4.19.0:`